### PR TITLE
Working with Wiki Suite API

### PR DESCRIPTION
**Pull request by ishifoev**
#80 #81 #82 #83

### CODE REVIEW

File: src/commands/wiki-api.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+  require_once(__DIR__ . '/../platform.php');
+  require_once(__DIR__ . '/helpers/services/getWiki.php');
+  require_once(__DIR__ . '/helpers/createSync.php');
+  require_once(__DIR__ . '/../database.php');
+  /*
+  E.g.: php src/cli-single.php  update-remote-service scoro
```
Comment (michielbdejong) [inline src/commands/wiki-api.php:7]: can you update this comment so it's about the wiki-api command?

File: src/database.php [hunk 2]
```diff
@@ -10,24 +10,28 @@ function getDbConn() {
   if (isset($test_db_connection)) {
     return $test_db_connection;
   } else {
-    return DriverManager::getConnection([
-      'url' => $_SERVER["DATABASE_URL"],
```
Comment (michielbdejong) [inline src/database.php:14]: Does this mean that specifying `DATABASE_URL` (as Heroku does) will no longer have any effect?

**Comment by michielbdejong** [comment]
Can you add some documentation about what this command does?

**Review by michielbdejong — APPROVED** [review]
Nice work!
So this command is for fetching data from a WikiSuite API and saving it to a file, right?
Please add this new command to https://prejournal.org/#commands to describe it, and then you can merge this PR.

**Comment by ishifoev** [comment]
> Can you add some documentation about what this command does?

Yes I will add this on documentation, I forget, because some working still in progress. But I will add.

**Comment by ishifoev** [comment]
Add new parser and write some test for exists JSON wiki suite.

**Comment by ishifoev** [comment]
Add Wiki API suite integration with prejournal insert inside movement with sync table.

**Pull request merged**